src/api/sendgrid.ts

```diff
@@ -0,0 +1,53 @@
+import mail, { MailDataRequired } from '@sendgrid/mail'
+import apiEnv from './apiEnv';
+
+export interface Personalizations {
+    to: [{
+        email: string,
+        name: string
+    }];
+    dynamic_template_data?: Record<string, string>;
+}
+
+mail.setApiKey(apiEnv.SENDGRID_API_KEY);
+
+const sendEmail = async (personalizations: Personalizations[], template_id: string) => {
+
+    console.log("SendGrid Mail Executed");
+
+    const message: MailDataRequired = {
+        personalizations: personalizations,
+        from: 'yizhang1728@gmail.com', // yuanjian@163
```

Comment:
use a fake email. you don't want your real email address to live in git history forever.